tests/python/topi/python/test_topi_depthwise_conv2d.py

```diff
@@ -268,7 +268,7 @@ def depthwise_conv2d_with_workload_NCHWc(batch, in_channel, in_height, channel_m
     filter_width = filter_height
     stride_h = stride_w = stride
 
-    assert dilation == 1, "depthwise_conv2d_NCHWc currently does not support dilation."
+    #assert dilation == 1, "depthwise_conv2d_NCHWc currently does not support dilation."
```

Comment:
If this is not useful anymore, please remove it rather than keeping it commented out here.

Reply:
Thanks for the review! The comments and logging have been removed!

Reply:
Sure, thanks for accepting the suggestions.